app/scaffold/pkgs/resolver.go

```diff
@@ -49,7 +51,24 @@ func (r *Resolver) Resolve(arg string, checkDirs []string) (path string, err err
 				Progress: os.Stdout,
 			})
 			if err != nil {
-				return "", fmt.Errorf("failed to clone repository: %w", err)
+				if err.Error() == "authentication required" {
```

Comment:
Should use the error comparison here

I think this is the one you're looking for

https://github.com/go-git/go-git/blob/d9b0a0dd2f4f03e82bcd7f7dad1f7b5eb42713c6/plumbing/transport/common.go#L34

See [This SO Issue](https://stackoverflow.com/questions/39121172/how-to-compare-go-errors) if you're unfamiliar.

Reply:
Oh! Awesome, this is actually new to me, thanks so much for the link

Reply:
Ah, it looks like plumbing/transport/common is actually internal, so I maybe can't import `ErrAuthenticationRequired`?